### DISCUSSION TITLE
Developer to Production

### DISCUSSION THREAD

**Discussion by lykaodan** [Budibase/budibase#3958] [Help]
Hello! I develop an app here in Budibase and is it possible to make developer machine into production machine? I want to deploy it. Should I use the import/export option? 

Hopefully someone can answer this, I really need this for our project. Thank you so much

**Comment by mjashanks** [comment]
The best thing to do is 
1. Setup your production instance.
2. Export all of your apps from the developer instance.
3. Import apps into your production instance.

There is no way to export/import settings at the moment.

**Reply by miltonbastos** [reply]
Hi! I've been getting to know Budibase for a few days. As I understand it, is it not possible to deploy the system autonomously? Does the system depend on having Budibase installed on the server?

Sorry about my bad english, I speak portuguese!

**Reply by mjashanks** [reply]
Hey @miltonbastos 

That's correct - Budibase apps need the whole Budibase stack to run. It's open-source of course, so you can run it anywhere that supports docker.
